services/ops/TopicWatcher/config

```diff
@@ -1,4 +1,15 @@
 {
+    "publish-settings":
+    {
+        "publish-local": false,
+        "publish-remote": true,
+        "remote":
```

Comment:
if there is a "remote" config it could imply that publish-remote is true. So why have a separate config "publish-remote"

Reply:
So you can turn it on or off regardless if the section is present or not